src/styles/variables.scss

```diff
@@ -7,13 +7,19 @@ $base-font-size: 16px;
 $font-sans: 'IBM Plex Sans', sans-serif;
 $font-mono: 'IBM Plex Mono';
 
+$color-white-alpha-15: rgba(255, 255, 255, 0.15);
+$color-white-alpha-30: rgba(255, 255, 255, 0.3);
+$color-white-alpha-50: rgba(255, 255, 255, 0.5);
+$color-white-alpha-70: rgba(255, 255, 255, 0.7);
+$color-white-alpha-75: rgba(255, 255, 255, 0.7);
```

Comment:
The idea of having something assigned to variables is that it might change over time. But it would make no sense to change any of those here. I can't imagine using `$color-white-alpha-70` with value of opacity set to for example `0.6`. So in my opinion it makes no sense to have these variables here.

If we want to save those variable at least the names should not contain values.

Also the alpha-75 is set to `0.7`.

And the last thing: I suggest to pass the color variables into the rgba function. Eg.: `rgba($color-white, 0.7);`

Reply:
Thanks for the detailed explanation @peronczyk 
Much appreciated!  also did all the changes you suggested.